.github/workflows/cd-pipeline.yml

```diff
@@ -129,3 +129,12 @@ jobs:
     with:
       draftReleaseReference: ${{ needs.create-release-notes.outputs.draftReleaseReference }}
 
+  deploy-to-prod:
+    needs: [publish-release-notes, create-and-post-tag]
```

Comment:
ditto

Reply:
ditto re he walked me through this in his POC